src/main/java/edu/ucsb/cs156/dining/ExampleApplication.java

```diff
@@ -14,7 +23,7 @@
  * The ExampleApplication class is the main entry point for the application.
  */
 @SpringBootApplication
-@Slf4j
+@Slf4j  //@EnableJpaAuditing(dateTimeProviderRef = "utcDateTimeProvider") doesn't work with mvn tests but works with backend
```

Comment:
Can we clean this up?

We don't put commented out code in PRs that are merged into main.